Python/pywarpx/picmi.py

```diff
@@ -1022,6 +1050,8 @@ class Simulation(picmistandard.PICMI_Simulation):
     # by Simulation objects to permit multiple WarpX runs simultaneously.
     extension = pywarpx.libwarpx
 
+    reduced_diagnostics = []
```

Comment:
This should be defined in the `init` method, as `self.reduced_diagnostics = []`.